src/main/java/teammates/ui/controller/StudentProfileEditSaveAction.java

```diff
@@ -52,7 +53,15 @@ private StudentProfileAttributes extractProfileData() {
         if ("".equals(editedProfile.nationality)) {
             editedProfile.nationality = getRequestParamValue("existingNationality");
         }
-        editedProfile.gender = getRequestParamValue(Const.ParamsNames.STUDENT_GENDER);
+
+        String value = Const.ParamsNames.STUDENT_GENDER;
```

Comment:
Are you trying to do `getRequestParamValue(Const.ParamsNames.STUDENT_GENDER)` instead? Otherwise this looks redundant.